toolkit/scripts/rpmops.sh

```diff
@@ -14,7 +14,21 @@ fi
 
 # Additional macros required to parse spec files.
 DIST_TAG=$(make -s -f $REPO_ROOT/toolkit/Makefile get-dist-tag)
-DEFINES=(-D "with_check 1" -D "dist $DIST_TAG")
+RELEASE_NUM=$(make -s -f $REPO_ROOT/toolkit/Makefile get-release-major)
+
+# Define 'azl <ver>' and 'azl<ver> 1' macros for use in spec files. Normally they would be defined in the 'mariner-rpm-macros-dist' package.
+# Dist tag will be of the form '.<distro><ver>', e.g. '.azl3'., rease number will be of the form '3.0'.
+
+# 'azl<ver> 1': Strip the leading '.' from the DIST_TAG
+distro_def="${DIST_TAG#.} 1"
+
+# 'azl <ver>': Strip the leading '.' and the trailing number from the DIST_TAG
+distro_name="$(echo ${DIST_TAG#.} | sed 's/[0-9]\+$//')"
+distro_majver="$(echo $RELEASE_NUM | cut -d'.' -f1)"
+distro_ver="${distro_name} ${distro_majver}"
+
+DEFINES=(-D "with_check 1" -D "dist $DIST_TAG" -D "$distro_def" -D "$distro_ver")
+echo "EXTRA DEFINES: ${DEFINES[*]}"
```

Comment:
Please don't echo anything extra to `stdout` for a successful run - this is meant to mimic `rpmspec` and is used by tooling parsing its output.

Reply:
Sorry, I had cleaned that up locally but my git upstream got messed up, re-pushed.